doc/changelogs/CHANGELOG_ARCHIVE.md

```diff
@@ -1,6 +1,7 @@
 # Node.js ChangeLog Archive
 
 <!--lint disable prohibited-strings-->
+<!--lint disable maximum-line-length-->
```

Comment:
I don't like that we're adding maintenance burden to these changelogs by requiring yet another linter comment.

Reply:
@gibfahn how bad do you think is this?

Reply:
@apapirovski If you prefer I can remove the comments `<!--lint disable maximum-line-length-->` in all the files inside `doc/changelogs` and add `.remarkrc` in the same folder with:
```
{
  "plugins": [
    ["../../tools/remark-preset-lint-node/node_modules/remark-lint-maximum-line-length", false]
  ]
}
```

Reply:
>@gibfahn how bad do you think is this?

Is it one per file? If so then nbd, we only create a new Changelog once per release line, and we usually start by copying the old one.

Reply:
Yes, it is per file.